src/dune/context.ml

```diff
@@ -35,6 +35,41 @@ module Env_nodes = struct
       (Dune_env.Stanza.find env_nodes.workspace ~profile).env_vars
 end
 
+let opam = Memo.Lazy.of_val (Bin.which ~path:(Env.path Env.initial) "opam")
+
+module Opam_config_var = struct
+  module Name = struct
+    include String
+
+    type t = string
```

Comment:
`module Name = String` perhaps? In fact, you can just use `String` directly without this alias.

Reply:
Sorry, this code is gone in the simplification.